test/metrics/npath/test_all_types.py

```
@@ -13,10 +14,14 @@
 
 
 def testIncorrectFormat():
-    with pytest.raises(Exception, match='PMDException'):
-        file = 'test/metrics/npath/ooo.java'
-        metric = NPathMetric(file)
-        metric.value(True)
+    file = 'test/metrics/npath/ooo.java'
+    metric = NPathMetric(file)
+    if shutil.which('mvn') is None:
+        with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)